PatientAuth/bin/src/main/resources/application.yml

```diff
@@ -0,0 +1,22 @@
+spring:
```

Comment:
Do we need separate authentication services for Doctor/Hospital or Patients.?

Reply:
For the initial sprints we need separate services for authentication of doctors and patients.. but in the end we will find a mechanism to separate doctors from patients and henceforth we will use a single service for authentication.